cmd/test.go

```diff
@@ -83,19 +85,23 @@ func validateArgs() error {
 	return nil
 }
 
-func RunTests() []*unversioned.FullResult {
-	fullResults := []*unversioned.FullResult{}
+func RunTests(out *output.OutWriter) bool {
+	pass := true
 	for _, file := range configFiles {
+		out.Banner(file)
 		tests, err := Parse(file)
 		if err != nil {
 			logrus.Fatalf("Error parsing config file: %s", err)
 		}
-		fullResults = append(fullResults, &unversioned.FullResult{
-			FileName: file,
-			Results:  tests.RunAll(),
-		})
+		results := tests.RunAll(out)
+		for _, result := range results {
+			if !result.Pass {
+				pass = false
+			}
+		}
+		pass = pass && out.FinalResults(results)
 	}
-	return fullResults
+	return pass
```

Comment:
just return out.FinalResults(results)

Reply:
I actually need to do this, because `FinalResults()` is only for a single config file; this boolean controls the exit code, so if one file passes but another fails, I want the end result to be a failure.